app/services/color/compare_hex.rb

```diff
@@ -1,6 +1,7 @@
 module Color
   class CompareHex
```

Comment:
At some point I'd love someone to sit with me and polish this "colors generator" a little so the output can be much more predictable. But so far, this is fine.